src/node/utils/Settings.js

```diff
@@ -441,6 +442,15 @@ function coerceValue(stringValue) {
       return false;
     }
 
+    if (stringValue === "null") {
+      /*
+       * There is already a way of setting a default value to null, and it is
+       * by using the syntax ${VAR_NAME}. This allows to use ${VAR_NAME:null} as
+       * well.
+       */
+      return null;
+    }
```

Comment:
I have split this modification in a dedicated commit (55dd44cc8048) and tried to document it.

However, I am not sure if it's better to include this or not, because it introduces an edge case (the string "null" would become special, and a crazy admin would no longer be able to call his user "null").

In theory, there is already complete syntactic support for "null" values without introducing special strings. See for example:

https://github.com/ether/etherpad-lite/blob/70990afd6680045b9887559b18e8a380aabc814c/settings.json.docker#L59-L66

On the other side, I understand that `${VAR_NAME:null}` is more explicit than `${VAR_NAME}`.

My questions for you are:
1. did you introduce this snippet because the documentation was not evident enough? In this case, I would prefer to drop this change & improve the documentation;
2. or did you do it because you were aware of the existing support but found it uncomfortable? In this case, let's discuss. I do not like the edge case, but user ergonomics is important.